references/validationFunctions/specimen/tumourGrade.js

```diff
@@ -11,43 +11,91 @@ const validation = ($row, $field) =>
       let codeList = [];
 
       switch ($row.tumour_grading_system && $row.tumour_grading_system.trim().toLowerCase()) {
-        case 'default':
+        case 'two-tier grading system':
           codeList = [
-            'gx - cannot be assessed',
-            'g1 well differentiated/low grade',
-            'g2 moderately differentiated/intermediated grade',
-            'g3 poorly differentiated/high grade',
-            'g4 undifferentiated/high grade',
+            'low grade',
+            'high grade',
           ];
           break;
-        case 'gleason':
+        case 'three-tier grading system':
           codeList = [
-            'gleason x: gleason score cannot be determined',
-            'gleason 2–6: the tumor tissue is well differentiated',
-            'gleason 7: the tumor tissue is moderately differentiated',
-            'gleason 8–10: the tumor tissue is poorly differentiated or undifferentiated',
+            'gx',
+            'g1',
+            'g2',
+            'g3',
           ];
           break;
-        case 'nottingham':
+        case 'four-tier grading system':
           codeList = [
-            'g1 (low grade or well differentiated)',
-            'g2 (intermediate grade or moderately differentiated)',
-            'g3 (high grade or poorly differentiated)',
+            'gx',
```

Comment:
just a small thing, I see these values used as the same codelist in a few places. maybe you could assign this to a variable and refer to it? 

i.e. before the switch statement you could have `const threeTierGradingList = ['gx','g1','g2','g3']`

then whereever it applies use `codeList = threeTierGradingList`

If you have to slightly the values later, it reduces the likelihood of a mistake